app/src/main/java/com/texthip/thip/ui/group/makeroom/viewmodel/GroupMakeRoomViewModel.kt

```
@@ -174,6 +298,59 @@ class GroupMakeRoomViewModel @Inject constructor(
+                            it.copy(
+                                isLoadingMoreSearchResults = false,
+                                isLastSearchPage = true
+                            )
+                        }
+                    }
+                }.onFailure {
+                    updateState {
+                        it.copy(isLoadingMoreSearchResults = false)
+                    }
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)